tests/integ/scala/test_async_job_suite.py

```diff
@@ -276,8 +279,13 @@ def test_multiple_queries(session, resources_path):
     )
     df = session.read.schema(user_schema).csv(f"@{tmp_stage_name1}/{test_file_csv}")
     assert len(df._plan.queries) > 1
-    res = df.collect_nowait()
-    Utils.check_answer(res.result(), df)
+    async_job = df.collect_nowait()
+    Utils.check_answer(async_job.result(), df)
+
+    # make sure temp object is dropped
```

Comment:
I added this additional check to make sure post actions run since I changed how we pass in post actions.